src/main/java/ch/njol/skript/expressions/ExprDurability.java

```diff
@@ -66,53 +70,73 @@ public String getPropertyName() {
 	}
 	
 	@Override
-	public Class<Short> getReturnType() {
-		return Short.class;
+	public Class<Object> getReturnType() {
```

Comment:
you aren't actually accepting Object.class, you're accepting a Number or a String